oslo_vmware/rw_handles.py

```diff
@@ -177,6 +177,32 @@ def read(self, chunk_size):
         """
         raise NotImplementedError()
 
+    def tell(self):
+        """Get the position of File Handle
+
+        :return: position
+        """
+        raise NotImplementedError()
+
+    def fileno(self):
+        """returns the integer file descriptor
+        by default this is not supported and raises IOError
+        """
+        raise IOError()
```

Comment:
`self._file_handle.fileno()` cannot be used? Why isn't this supported?

Reply:
This particular behaviour was inspired from here:
https://github.com/sapcc/cinder/blob/stable/queens-m3/cinder/backup/chunkeddriver.py#L677

Reply:
OK, then just mention it.